apps/openmw/mwgui/windowmanagerimp.cpp

```diff
@@ -2083,6 +2088,25 @@ namespace MWGui
         SDL_free(text);
     }
 
+    void WindowManager::toggleConsole()
+    {
+        bool visible = !mConsole->isVisible();
```

Comment:
"visible = not visible"

The logic of this variable is a bit confusing and may hurt maintainability.